webpack.config.js

```diff
@@ -113,7 +113,8 @@ module.exports = {
     alias: {
       css: path.join(__dirname, "/troposphere/static/css/"),
       images: path.join(__dirname, "/troposphere/static/images/"),
-      highcharts: "highcharts-commonjs"
+      highcharts: "highcharts-commonjs",
+      react: path.join(__dirname, "/node_modules/react")
```

Comment:
hrmm.. I don't think this is the solution. We need to figure out why extra versions of react are getting bundled. Can you post the error? How would multiple versions of React get bundled? `troposphere-ui` doesn't include `React`.